homeassistant/components/buienradar/camera.py

```diff
@@ -60,7 +60,13 @@ class BuienradarCam(Camera):
     """ Deadline for image refresh """
     _deadline = None        # type: Optional[datetime]
     _name = ""              # type: str
-    _delta = 0.0         # type: float
+    _delta = 600.0          # type: float
+    """
+    Ensures that only one reader can cause an http request at the same time,
```

Comment:
This should be a comment instead, using `#`.

Reply:
I thought docstrings need the `"""` style. I just found out that docstrings should be below the variable though in many style guides (e.g. [google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)). Will move this one and the one on line 60 when I get to it (likely: tomorrow night)

Reply:
Yes docstring need `"""` and have to be put right under the class or function. Variables don't have their own docstrings.

Reply:
They seem to have their own docstrings in the styleguide (see below). On the other hand, sphinx/pydoc do not seem to pick this up so I'm fine with changing it to a comment.
```py
    def __init__(self, param1, param2, param3):
# ...
        self.attr3 = param3  #: Doc comment *inline* with attribute

        #: list of str: Doc comment *before* attribute, with type specified
        self.attr4 = ['attr4']

        self.attr5 = None
        """str: Docstring *after* attribute, with type specified."""
```

Reply:
Switched style since I do not hold a strong preference one way or the other and the style of comment is looser: No first line summary + whitespace + body required..